### ISSUE TITLE
FID

### ISSUE THREAD

**Issue by diamond0910** [ziqihuangg/Collaborative-Diffusion#30]
Hi, I want to ask how you calculate the FID. Generating 3000 image for the 3000 testing data and calculate the FID between these 3000 images and the training dataset or the whole dataset?

**Comment by diamond0910** [comment]
I calculate FID between the 3000 generated images with the whole dataset, and I get 17.27, which is different with 111.36 of the paper. What is wrong?

**Comment by ziqihuangg** [comment]
Thanks for the question. To ensure fair comparison with TediGAN, we used 50 test images from the test set. A larger set of test images leads to a better (lower) FID, and that’s why when you test on 3k images, the FID is much lower. If you simply want to compare with Collaborative Diffusion, using 3000 images' FID will give you more reliable results.

**Comment by diamond0910** [comment]
> Thanks for the question. To ensure fair comparison with TediGAN, we used 50 test images from the test set. A larger set of test images leads to a better (lower) FID, and that’s why when you test on 3k images, the FID is much lower. If you simply want to compare with Collaborative Diffusion, using 3000 images' FID will give you more reliable results.

Thank you for your reply. I am following your work and will soon cite it. Could you please send me a pretrained model of TediGAN and Composable on your training set for me to display some visual results? If possible, I would greatly appreciate it. My email is 576825820@qq.com.

**Comment by gaoyixuan111** [comment]
我正在关注你的工作，很快就会引用它。您能否在您的训练集上向我发送 TediGAN 和 Composable 的预训练模型，以便我显示一些视觉结果？如果可以的话，我将不胜感激。我的邮箱是gyx18336058726@163.com

**Comment by ziqihuangg** [comment]
For Composable, the pre-trained models are the same text-to-face and mask-to-face models we provided in this repo. To combine these pre-trained models over sampling, please refer to Composable Diffusion's official implementation.

For TediGAN, please refer to their official release.
